### PR TITLE
Add glibc error

### PR DESCRIPTION
**Pull request by juli1**
### What does this PR do? What is the motivation?

This PR adds more error cases customers had. Customers will find the resolution to these issues in the page instead of asking a customer success manager.

### CODE REVIEW

File: content/en/code_analysis/troubleshooting/_index.md [hunk 1]
```diff
@@ -44,6 +44,28 @@ For performance issues, include the following information:
...
+
+If you run the static analyzer in your CI pipeline and get an error message similar to the following line:
+
+```
+version `GLIBC_X.YY' not found
+```
+
+It means that you are either:
+
+- running your CI pipeline with a Linux distribution that contains an old version of the glibc. In this case, we
```
Comment (michaelcretzman) [inline content/en/code_analysis/troubleshooting/_index.md:57]:
```suggestion
- running your CI pipeline with a Linux distribution that contains an old version of the glibc. In this case,
```

File: content/en/code_analysis/troubleshooting/_index.md [hunk 2]
```diff
@@ -44,6 +44,28 @@ For performance issues, include the following information:
...
+  recommend you upgrade to the latest version. Our analyzer will always run with the latest of Ubuntu/Debian based-systems.
```
Comment (michaelcretzman) [inline content/en/code_analysis/troubleshooting/_index.md:58]:
```suggestion
  recommend you upgrade to the latest version. The analyzer will always run with the latest of Ubuntu/Debian based-systems.
```

File: content/en/code_analysis/troubleshooting/_index.md [hunk 3]
```diff
@@ -44,6 +44,28 @@ For performance issues, include the following information:
...
+- running your CI pipeline with a Linux distribution that does not rely on the glibc (such as Alpine Linux). We recommend
```
Comment (michaelcretzman) [inline content/en/code_analysis/troubleshooting/_index.md:59]:
```suggestion
- running your CI pipeline with a Linux distribution that does not rely on the glibc (such as Alpine Linux). Instead,
```

File: content/en/code_analysis/troubleshooting/_index.md [hunk 4]
```diff
@@ -44,6 +44,28 @@ For performance issues, include the following information:
...
+  you run your CI pipeline with a distribution that supports the latest version of the glibc (such as the stable version of Ubuntu).
+
+### Results are not being surfaced in the Datadog UI
+
+If you are running on a non-GitHub repository, make sure you first commit on your default branch (e.g. branch name like
```
Comment (michaelcretzman) [inline content/en/code_analysis/troubleshooting/_index.md:64]:
```suggestion
If you are running on a non-Github repository, ensure you first commit on your default branch (for example, a branch name like
```

File: content/en/code_analysis/troubleshooting/_index.md [hunk 5]
```diff
@@ -54,6 +76,13 @@ For issues with Datadog Software Composition Analysis, include the following inf
 - The name of the branch you ran the analysis on
 - The list of dependency files in your repository (such as `package-lock.json`, `requirements.txt`, or `pom.xml`)
 
+### Results are not being surfaced in the Datadog UI
+
+If you are running on a non-GitHub repository, make sure you first commit on your default branch (e.g. branch name like
```
Comment (michaelcretzman) [inline content/en/code_analysis/troubleshooting/_index.md:81]:
```suggestion
If you are running on a non-Github repository, ensure you first commit on your default branch (for example, a branch name like
```

**Review by michaelcretzman — APPROVED** [review]
Approved with minor suggestions.

File: content/en/code_analysis/troubleshooting/_index.md [hunk 6]
```diff
@@ -44,6 +44,28 @@ For performance issues, include the following information:
...
+If you run the static analyzer in your CI pipeline and get an error message similar to the following line:
+
+```
+version `GLIBC_X.YY' not found
+```
+
+It means that you are either:
+
+- running your CI pipeline with a Linux distribution that contains an old version of the glibc. In this case,
+  recommend you upgrade to the latest version. The analyzer will always run with the latest of Ubuntu/Debian based-systems.
```
Comment (kassenq) [inline content/en/code_analysis/troubleshooting/_index.md:58]: "In this case, Datadog recommends upgrading to the latest version."

File: content/en/code_analysis/troubleshooting/_index.md [hunk 7]
```diff
@@ -54,6 +76,13 @@ For issues with Datadog Software Composition Analysis, include the following inf
 - The name of the branch you ran the analysis on
 - The list of dependency files in your repository (such as `package-lock.json`, `requirements.txt`, or `pom.xml`)
 
+### Results are not being surfaced in the Datadog UI
+
+If you are running on a non-Github repository, ensure you first commit on your default branch (for example, a branch name like
```
Comment (kassenq) [inline content/en/code_analysis/troubleshooting/_index.md:81]: "If you are running Code Analysis on a non-GitHub repository, ensure that the first scan is ran on your default branch ...."

**Review by kassenq — CHANGES_REQUESTED** [review]

File: content/en/code_analysis/troubleshooting/_index.md [hunk 8]
```diff
@@ -44,6 +44,27 @@ For performance issues, include the following information:
...
+
+```
+version `GLIBC_X.YY' not found
+```
+
+It means that you are either:
+
+- running your CI pipeline with a Linux distribution that contains an old version of the glibc. In this case, Datadog recommends upgrading to the latest version. The analyzer always runs with the latest of Ubuntu/Debian based-systems.
+- running your CI pipeline with a Linux distribution that does not rely on the glibc (such as Alpine Linux). Instead,
+  you run your CI pipeline with a distribution that supports the latest version of the glibc (such as the stable version of Ubuntu).
```
Comment (jacobotb) [inline content/en/code_analysis/troubleshooting/_index.md:59]:
```suggestion
  run your CI pipeline with a distribution that supports the latest version of the glibc (such as the stable version of Ubuntu).
```

**Review by jacobotb — APPROVED** [review]

**Review by kassenq — APPROVED** [review]

**Pull request merged**